sdk/communication/communication-email/swagger/README.md

```diff
@@ -21,5 +21,20 @@ azure-arm: false
 add-credentials: false
 v3: true
 use-extension:
-  "@autorest/typescript": "6.0.0-dev.20200623.2"
+  "@autorest/typescript": "6.0.0-beta.15"
```

Comment:
Would you mind trying to re-generate with `6.0.0-rc.1`? It is the latest version